core/src/main/java/com/google/cloud/sql/core/SqlAdminApiFetcher.java

```diff
@@ -152,14 +151,22 @@ public InstanceData getInstanceData(
                             .orElse(x509Certificate.getNotAfter());
                   }
 
+                  logger.info(
+                      String.format(
+                          "[%s %d] INSTANCE DATA DONE",
+                          instanceName, Thread.currentThread().getId()));
```

Comment:
Let's leave the thread ID off -- unless it's worth keeping?

Reply:
ditto with below

Reply:
Fixed.